main.rb

```diff
@@ -0,0 +1,186 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+class App
+  def initialize
+    @people = []
+    @books = []
+  end
+
+  def option_chosen(option)
+    case option
+    when '1'
+      all_books
+    when '2'
+      all_people
+    when '3'
+      make_person
+    when '4'
+      make_book
+    when '5'
+      rent_book
+    when '6'
+      list_rents
+    else
+      wrong_option
+    end
+  end
+
+  def print_book(book, index = nil)
+    "#{index + 1}) #{book.title} by #{book.author}"
+  end
+
+  def all_books
+    puts 'Books List: '
+    @books.each_with_index { |book, index| puts print_book(book, index) }
+    return 'No books' if @books.empty?
+  end
+
+  def print_person(person, index = 1)
+    "#{index + 1}) [#{person.class.name}] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+  end
+
+  def all_people
+    puts 'People List: '
+    @people.each_with_index { |person, index| puts print_person(person, index) }
+    return 'No people' if @people.empty?
+  end
+
+  def make_person
+    print 'Do you want to create a student (1) or a teacher (2)? '
+    opt = gets.chomp
+    return wrong_option unless %w[1 2].include? opt
+
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    return create_student(age.to_i, name) if opt == '1'
+    return create_teacher(age.to_i, name) if opt == '2'
+  end
+
+  def create_student(age, name)
+    print 'Has Parent Permission? [Y/N]: '
+    pp = gets.chomp
+    return wrong_option unless %w[y n].include? pp.downcase
+
+    @people << Student.new(age, name, parent_permission: pp.downcase == 'y')
+    'Student created successfully'
+  end
```

Comment:
- [ ] When creating a student the output is not correct as you can see in the next image:

![Screen Shot 2022-03-03 at 19 53 54](https://user-images.githubusercontent.com/67211919/156684441-4d2e33fc-ae79-4aa3-a6b4-fec230301f7b.png)